qb_upload_limit_per_day.py

```diff
@@ -19,12 +19,92 @@
 current_session_previous_day_upload_data_usage = 0.0
 qb_online_status = False
 
+def login():
+    """
+    Performs login to qBittorrent WebUI using data from secrets.json
+    If login is successful, the resulting cookies are stored in 
+    cookies.txt file, so that the other methods can access them 
```

Comment:
```suggestion
    cookies.json file, so that the other methods can access them 
```
The contents are json so let that be a json file.